components/CSS/component.css

```diff
@@ -7,34 +7,35 @@
     background-color: var(--color-primary);
     color: var(--white);
     font-family: var(--font-dosis);
+    position: sticky;
+    top: 0;
+    align-self: flex-start;
+    z-index: 9999;
 }   
 
 .content-box {
     display: flex;
+    width: 100%;
+    height: 100vh;
     flex-wrap: nowrap;
     
 }
 .side-container {
```

Comment:
for sidebar scroll there is some issue i've noticed if you don't scroll main content scroll bar the sidebar content is not visible,
one thing i would suggest you to give some height for sidebar div so it may be fix that